src/routes/articles.js

```diff
@@ -0,0 +1,26 @@
+import express from 'express';
+import ArticleController from '../controllers/article';
+import verifySlug from '../middlewares/verifySlug';
+import { verifyToken, checkUser } from '../middlewares/authentication';
+import { checkInput } from '../middlewares/validateArticle';
+import { paramsChecker } from '../middlewares/checkParams';
+
+const articlesRouter = express.Router();
+
+const {
+  createArticle,
+  getAllArticles,
+  updateArticle,
+  getArticle,
+  deleteArticle
+} = ArticleController;
+
+const { slugChecker } = verifySlug;
+
+articlesRouter.get('/articles', verifyToken, getAllArticles);
```

Comment:
Here, should all users (both registered and not registered) be able to see the list of articles? Don't you think authentication, and by extension, the `verifyToken` middleware, is not required?

Same goes for `articlesRouter.get('/articles/:slug' ...` (the next line).